src/util/Authentication.js

```diff
@@ -0,0 +1,38 @@
+function setCookie(cName, value, days){
+    const date = new Date();
+    date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000))
+    const expires = "expires=" + date.toUTCString();
+    document.cookie = `${cName}= ${value}; ${expires}; "path=/"`
+}
+
+function getCookie(cName) {
+  const cDecoded = decodeURIComponent(document.cookie);
+  const cArray = cDecoded.split( "; ");
+  let result = null;
+
+  cArray.forEach(element => {
+    if(element.indexOf(cName) === 0){
+        result = element.substring(cName.length + 1)
+    }
+  })
+  return result;
+}
+
+function deleteCookie(cName){
+    setCookie(cName, null, null)
+}
```

Comment:
All three functions work great. Nice job. Can you export each of these from the file?

Reply:
These still need to be exported from the file using the **export** keyword.